test/e2e/map-delete-action.test.js

```diff
@@ -0,0 +1,56 @@
+const {assert, getDriver} = require('vl-ui-core').Test.Setup;
+const VlMapDeleteActionPage = require('./pages/vl-map-delete-action.page');
+
+describe('vl-map-delete-action', async () => {
+  let vlMapPage;
+  let driver;
+
+  beforeEach(() => {
+    driver = getDriver();
+    vlMapPage = new VlMapDeleteActionPage(getDriver());
+    return vlMapPage.load();
+  });
+
+  it('als gebruiker zie ik dat de delete actie actief staat', async () => {
+    const deleteAction = await vlMapPage.getDeleteAction();
+    await assert.eventually.isTrue(deleteAction.isActive());
+  });
+
+  it('kan features deleten door er op te klikken', async () => {
+    const deleteAction = await vlMapPage.getDeleteAction();
+    const layer = await deleteAction.getLayer();
+    await driver.wait(async () => {
+      const aantalFeatures = await layer.getNumberOfFeatures();
+      return aantalFeatures == 3;
+    });
```

Comment:
Misschien proberen dit te verwerken in het page object. Initieel wachten tot het aantal features gekend is en na verwijderen wachten tot het aantal features verminderd is met 1. Op die manier moet er geen wait geïmplementeerd worden bij de afnemers.

Reply:
Het lijkt erop dat bovenstaande code dient om het aantal features te checken. Aangepast door await assert.eventually.equal(....).

Reply:
Als dat het doel is, zou het `await driver.wait` deel niet nodig zijn.

Reply:
Gaat assert eventually niet gewoon wachten tot de promise resolved is? En niet tot "wat de promise teruggeeft waar is"?